Registry/RegistryParse.py

```diff
@@ -273,6 +290,38 @@ def last_hbin_offset(self):
         """
         return self.unpack_dword(0x28)
 
+    def calculate_checksum(self, include_checksum = False):
+        """
+        checksum is calculated over the first 0x200 bytes
+        XOR of all D-Words from 0x00000000 to 0x000001FB
+        If include_checksum is True, zero is the expected return value for an valid hive.
```

Comment:
Thin of The value at 0x1FC as a "filler"
The sum (xor) of all the dwords up to and including the filler should be zero
So summing them all should be easier to understand if it is only for checking that it is valid.
Would it make more sense to call the parameter `expect_zero`

`checksum2` does not make sense to me since it is only one checksum.
But to make it more understandable I will revert to using only 0x1FB and `return self.calculate_checksum() == self.checksum()`

The reason for the checksum fetch is for "dissecting" and in part forensics on reg files. While the calculation with non zero result is to be able to write the correct value in a corrupt or new hive file.